### PR TITLE
Add Wiz Icon

### LINKED ISSUE #10612: Request: Wiz
### Brand Name

Wiz

### Website

https://wiz.io

### Popularity Metric

The Similarweb [rank](https://www.similarweb.com/website/wiz.io/) is 141,495

### Official Resources for Icon and Color

WizStar24

wiz-logo-official

color: #0254EC

### Additional Comments

There are two variants of the logo, the recognized one is the simple star, and the full-fledged
one is contains the brand name: Wiz.

### PR DESCRIPTION
**Pull request by MinnThikeTunn**
wiz (3)

**Issue:** closes #10612 

**Popularity metric:**  [~141.3k](https://www.similarweb.com/website/wiz.io/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
-  [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I found two variants according to issues. I didn't know anything about handling two variants and didn't see anything about how I deal with logo variants in contributing guidelines. So, according to the issue creator, I choose the recognized one. I am sorry if I made some mistakes.

//Edited
Since it is my first time, I am sorry that I caused any trouble.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -17134,6 +17134,11 @@
             "source": "https://www.wix.com/about/design-assets",
             "guidelines": "https://www.wix.com/about/design-assets"
         },
+        {
+            "title": "Wiz",
+            "hex": "0254EC",
+            "source": "https://www.wiz.io"
```
Comment (uncenter) [inline _data/simple-icons.json:17140]:
```suggestion
            "source": "https://www.wiz.io/newsroom#media-kit"
```

File: icons/wiz.svg
Comment (uncenter) [inline icons/wiz.svg:1]: This isn't matching up with any of the logos from the [press kit](https://www.wiz.io/newsroom#media-kit), nor is it matching the one from the header (which is different then the press kit ones for some reason).

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/wiz.svg
Comment (MinnThikeTunn) [reply icons/wiz.svg:1]: Yeah,I will replace this with an appropriate press kit.

Comment (adamrusted) [reply icons/wiz.svg:1]: Hey @MinnThikeTunn - are you still working on this?

**Comment by adamrusted** [comment]
Closing this PR, given it's not had updates from the original creator  for some time. Will un-link from the issue so we can work on the issue fresh, should another contributor want to.

**Pull request closed without merging**